### DISCUSSION TITLE
Pytorch 高级篇（4）：图像标注（Image Captioning (CNN-RNN)） | SHEN's BLOG

### DISCUSSION THREAD

**Discussion by veraposeidon** [veraposeidon/veraposeidon.github.io#93] [General]
https://shenxiaohai.me/2018/10/22/pytorch-tutorial-advanced-04/

**Comment by kukuhaza** [comment]
你好，我在下载数集的时候，$ ./download.sh运行这个的时候出现错误，一直没改对，请问可以帮忙看看怎么改吗？十分感谢！！
错误是download.sh: line 7: syntax error near unexpected token `newline' 
                download.sh: line 7: `<!DOCTYPE html>'

**Comment by veraposeidon** [comment]
应该是换行符的问题，你可以在Linux下拿文本编辑器重新编辑download.sh里的换行，大概意思就是两行变一行然后再换行。

不行的话，你就打开download.sh，然后一条一条指令运行好了，一样的效果。

**Comment by kukuhaza** [comment]
好的谢谢  我先试试

**Comment by kukuhaza** [comment]
以顺利解决。。。
请问这个模型的精确度您计算过吗？？
还有怎么能任意输入一张图片来进行测试呢？
谢谢

**Comment by kukuhaza** [comment]
你好 我在测试的过程中 出现了以下问题
File ”.......PIL/Image.py ,line2764, in open fp=builtins.open(filename,”rb”)
IOError:[Error 2] No such file or directory :’png/football2.jpg

我打开Image.py import __builtin__部分好像出错了，但是一直没改成功，请问可以给解答一下吗？

**Comment by veraposeidon** [comment]
@kukuhaza 
1. 精确度没有计算过，可能需要你去调研一下。
2. Error应该是提示没有图片吧，说明路径不对。另外，这是一份博客，源码对应的材料和文件目录层次可以查看[我的Github](https://github.com/veraposeidon/pytorch-tutorial/tree/master/tutorials/03-advanced/image_captioning)。确认下是否都对应上了。

**Comment by kukuhaza** [comment]
好的谢谢，错误已改正！十分感谢您的分享！

**Comment by veraposeidon** [comment]
嗯，不客气。
